busmall.js

```diff
@@ -176,6 +178,30 @@ function renderChart() {
       }
     }
   });
+
+  var button = document.createElement('button');
+  button.textContent = 'Vote Again';
+  divEl.appendChild(button);
+  button.addEventListener('click', function reload() {
+    button.removeEventListener('click', reload);
+    location.reload();
+  });
+}
+
+function saveData() {
+  if (localStorage.total) {
+    localStorage.setItem('current', JSON.stringify(Item.totalVotes));
+    var parseCurrent = JSON.parse(localStorage.current);
```

Comment:
Since you do not use `localStorage.current` anywhere else in your code, you could do this without storing `Item.totalVotes`.  Just use `Item.totalVotes` in your `for` loop instead of `parseCurrent`.